plugin.py

```diff
@@ -29,9 +30,26 @@ def is_allowed_to_start(
     ) -> Optional[str]:
         if not workspace_folders:
             return "Requires a folder to start."
-        path = workspace_folders[0].path
-        tailwind_config_file_path = os.path.join(path, 'tailwind.config.js')
-        if not os.path.exists(tailwind_config_file_path):
-            return "No tailwind.config.js present in {}".format(path)
-        return None
 
+        # Config pattern is found here:
+        # https://github.com/tailwindlabs/tailwindcss-intellisense/blob/766a5d533dcb68640ce6b3270488f6701dd1173d/packages/vscode-tailwindcss/src/extension.ts#L40
+        config_file_pattern = '(tailwind|tailwind.config).(js|cjs)'
+        folder_exclude_patterns = sublime.load_settings('Preferences.sublime-settings').get("folder_exclude_patterns", []) # type: List[str]
+        folder_exclude_patterns.append('node_modules') # definitely exclude node_modules
```

Comment:
Converting to `Set` would result in faster lookup and avoid potential duplicates.